inductiva/_cli/cmd_tasks/download.py

```diff
@@ -0,0 +1,56 @@
+"""Downloads a tasks by id via CLI."""
+import argparse
+import inductiva
+
+
+def download_task(args):
+    """Downloads a task by id."""
+    ids = args.id
+    filenames = args.filenames
+    output_dir = args.output_dir
+
+    if not ids:
+        print("No id(s) specified.\n"
+              "> Use `inductiva tasks download -h` for help.")
+        return 1
+
+    ids = set(ids)
+
+    if len(ids) > 1 and output_dir is not None:
+        print("Cannot specify `output_dir` for more than one download")
+        return 1
+
+    for task_id in ids:
```

Comment:
Be consistent with the names, if here one uses `task_id` than the set should be `task_ids` or `task_ids_list`.